From 8e6df2e500ddbf08feaeb2123fc5b1ce420fa15a Mon Sep 17 00:00:00 2001
From: Loc Vu <loc.vu.zn@renesas.com>
Date: Tue, 30 Mar 2021 15:37:30 +0700
Subject: [PATCH] arm64: dts: renesas: r9a07g044l2-smarc: Fix compatible of
 vccq_sdhi0

VCCQ of SDHI0 is controlled by GPIO so that correct type of regulator
from fixed to gpio.
Also update value of status property to be consistent with the comment.
User just need to comment out the eMMC to use uSD.
No need to update status anymore.

Signed-off-by: Loc Vu <loc.vu.zn@renesas.com>
---
 arch/arm64/boot/dts/renesas/r9a07g044l2-smarc.dts | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/arch/arm64/boot/dts/renesas/r9a07g044l2-smarc.dts b/arch/arm64/boot/dts/renesas/r9a07g044l2-smarc.dts
index ecd4943..89c2998 100644
--- a/arch/arm64/boot/dts/renesas/r9a07g044l2-smarc.dts
+++ b/arch/arm64/boot/dts/renesas/r9a07g044l2-smarc.dts
@@ -130,7 +130,7 @@
 	};
 
 	vccq_sdhi0: regulator-vccq-sdhi0 {
-		compatible = "regulator-fixed";
+		compatible = "regulator-gpio";
 
 		regulator-name = "SDHI0 VccQ";
 		regulator-min-microvolt = <1800000>;
@@ -587,7 +587,7 @@
 	bus-width = <4>;
 	sd-uhs-sdr50;
 	sd-uhs-sdr104;
-	status = "disabled";
+	status = "okay";
 };
 */
 
-- 
2.7.4

